static/app/views/replays/details.tsx

```diff
@@ -113,17 +105,18 @@ function ReplayDetails() {
             </Panel>
           </ReplayLayout>
           <Layout.Side>
-            <UserActionsNavigator event={event} entry={breadcrumbEntry} />
+            <UserActionsNavigator
+              event={replay.getEvent()}
+              entry={replay.getEntryType(EntryType.BREADCRUMBS)}
+            />
           </Layout.Side>
           <Layout.Main fullWidth>
             <Panel>
-              <BreadcrumbTimeline crumbs={breadcrumbEntry?.data.values || []} />
+              <BreadcrumbTimeline
+                crumbs={replay.getEntryType(EntryType.BREADCRUMBS)?.data.values || []}
```

Comment:
This is less of an issue because it's memoized, but we call `getEntryType` multiple times for breadcrumbs.

Reply:
Actually I wanted to follow up again and figure out how to change the props for all these things so the `replay` instance gets passed down lower. That instance has important things like the start/end timestamps. We need to use those values instead of the timestamp of the first/last breadcrumb.